cmd/containerd-shim-runhcs-v1/computeagent.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	context "context"
+	"fmt"
+	"net"
+
+	"github.com/Microsoft/go-winio"
+	"github.com/Microsoft/hcsshim/internal/computeagent"
+	"github.com/Microsoft/hcsshim/internal/hns"
+	"github.com/Microsoft/hcsshim/internal/uvm"
+	"github.com/Microsoft/hcsshim/pkg/octtrpc"
+	"github.com/containerd/ttrpc"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Microsoft/hcsshim/internal/log"
+)
+
+// This file holds the implementation of the Compute Agent service that is exposed for
+// external network configuration.
+
+// %s is Pod ID
```

Comment:
I don't feel like this comment is necessary. Maybe instead you could rename the const to  `podComputeAgentFmt`  or something if you want it to be more clear.